file-formats/edcc/type/zif_aff_edcc_v1.intf.abap

```diff
@@ -0,0 +1,552 @@
+      "! Additional data source
+      data_source             TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Additional Description</p>
+      "! Additional check description
+      "! $required
+      additional_description  TYPE zif_aff_types_v1=>ty_description_60,
+      "! <p class="shorttext">Comparison Field Groups</p>
+      "! Comparison field groups
+      "! $required
+      comparison_field_groups TYPE ty_comparison_field_groups,
+    END OF ty_content_check.
+  "! <p class="shorttext">Content Checks</p>
+  "! Assigned content checks
+  TYPES ty_content_checks    TYPE SORTED TABLE OF ty_content_check WITH UNIQUE KEY check_id check_number.
+  "! <p class="shorttext">Sequence</p>
+  "! Check run sequence
+  TYPES ty_run_sequence      TYPE c LENGTH 3.
+  "! <p class="shorttext">Control Parameter</p>
+  "! Check execution control parameter
+  "! $values {@link zif_aff_edcc_v1.data:co_control_parameter}
+  "! $default {@link zif_aff_edcc_v1.data:co_control_parameter.always_run_check}
+  TYPES ty_control_parameter TYPE c LENGTH 2.
+  TYPES:
+    "! <p class="shorttext">Run Attribute</p>
+    "! Check run attribute
+    BEGIN OF ty_run_attribute,
+      "! <p class="shorttext">Check ID</p>
+      "! Check id
+      "! $required
+      check_id          TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Sequence</p>
+      "! Check run sequence
+      "! $required
+      run_sequence      TYPE ty_run_sequence,
+      "! <p class="shorttext">Control Parameter</p>
+      "! Check execution control parameter
+      control_parameter TYPE ty_control_parameter,
+    END OF ty_run_attribute.
+  "! <p class="shorttext">Run Attributes</p>
+  "! Check run attributes
+  TYPES ty_run_attributes TYPE SORTED TABLE OF ty_run_attribute WITH UNIQUE KEY check_id.
+  TYPES:
+    "! <p class="shorttext">Representation Type</p>
+    "! Assigned representation type
+    BEGIN OF ty_representation_type,
+      "! <p class="shorttext">Name</p>
+      "! Assigned representation type
+      "! $required
+      name TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_representation_type.
+  "! <p class="shorttext">Representation Types</p>
+  "! Assigned representation types
+  TYPES ty_representation_types TYPE SORTED TABLE OF ty_representation_type WITH UNIQUE KEY name.
+  TYPES:
+    "! <p class="shorttext">Checks</p>
+    "! Assigned checks
+    BEGIN OF ty_checks,
+      "! <p class="shorttext">Existence Check</p>
+      "! Assigned existence check
+      "! $required
+      existence_check TYPE ty_existence_check,
+      "! <p class="shorttext">Status Checks</p>
+      "! Assigned status checks
+      status_checks   TYPE ty_status_checks,
+      "! <p class="shorttext">Content Checks</p>
+      "! Assigned content checks
+      content_checks  TYPE ty_content_checks,
+      "! <p class="shorttext">Run Attributes</p>
+      "! Check run attributes
+      run_attributes  TYPE ty_run_attributes,
```

Comment:
Minor: Just as information. You can just resolve this comment. 

I preferred my suggestion with `sequence` or `check_sequence`. I guess for the user it becomes more clear instead of "run attributes".

However, this is your decision. I just want to share my feedback. :)